src/login/loginctl.c

```diff
@@ -1017,11 +988,7 @@ static int activate(int argc, char *argv[], void *userdata) {
 
         for (i = 1; i < argc; i++) {
 
-                r = sd_bus_call_method(
-                                bus,
-                                "org.freedesktop.login1",
-                                "/org/freedesktop/login1",
-                                "org.freedesktop.login1.Manager",
+                r = bus_call_method(bus, &login_mgr,
```

Comment:
It'd be nicer to move "bus, &login_mgr," to a new line and indent it the same as the following arguments.